### PR TITLE
Excel report generation

### PR DESCRIPTION
**Pull request by Shurron**
Completed #20, #21

### CODE REVIEW

File: src/QuizData.Analyser/Models/NumericDistribution.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+using System.Linq;
+
+namespace QuizData.Analyser.Models
+{
+    /// <summary>
+    /// Represents number distribution
+    /// </summary>
+    public class NumericDistribution
+    {
+        public List<NumericDistributionPart> Parts { get; set; }
```
Comment (vkamiansky) [inline src/QuizData.Analyser/Models/NumericDistribution.cs]: Advised changing to "intervals" a more commonly used term.

File: src/QuizData.ExcelReporter/ExcelReporter.cs [hunk 2]
```diff
@@ -0,0 +1,241 @@
+﻿using System.IO;
+using System.Linq;
+using OfficeOpenXml;
+using QuizData.Analyser.Models;
+using System.Collections.Generic;
+using OfficeOpenXml.Drawing.Chart;
+
+namespace QuizData.ExcelReport
+{
+    public class ExcelReporter
```
Comment (vkamiansky) [inline src/QuizData.ExcelReporter/ExcelReporter.cs:10]: Needs a test coverage. md5 checksum advised.

**Review by vkamiansky — CHANGES_REQUESTED** [review]
Some minor changes needed. Otherwise ready.

File: src/QuizData.Analyser/Models/NumericDistribution.cs (hunk 1 above)
Comment (Shurron) [reply src/QuizData.Analyser/Models/NumericDistribution.cs]: [NumericDistributionPart renamed to NumericDistributionInterval](https://github.com/vkamiansky/analysis-practice-jul-2019/commit/662accd56b3e6ff3d86a41b8863d21f044f117cb). В `DoubleNumericDistribution` не стал ничего менять, так как `DoubleNumericDistributionPart` уже не похоже на интервал, скорее на площадь.

File: src/QuizData.ExcelReporter/ExcelReporter.cs (hunk 2 above)
Comment (Shurron) [reply src/QuizData.ExcelReporter/ExcelReporter.cs:10]: Completed in [Added one simple unit test for ExcelReporter](https://github.com/vkamiansky/analysis-practice-jul-2019/commit/c9bb20593e1bc9192a8276e26a4d5fee43d368a4)

**Review by vkamiansky — APPROVED** [review]

**Pull request merged**